Day2/2b_exercise_optimization2.ipynb

```diff
@@ -1,3 +1,4 @@
+# my first pull request 
```

Comment:
This probably breaks the notebook :)

Can you undo this change and instead modify the README.md. For example, add a "Tobias was here" note to the bottom.